src/rpc/blockchain.cpp

```diff
@@ -3,24 +3,24 @@
 // Distributed under the MIT/X11 software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include "main.h"
-#include "rpcserver.h"
-#include "rpcprotocol.h"
-#include "init.h" // for pwalletMain
-#include "block.h"
-#include "checkpoints.h"
-#include "txdb.h"
-#include "neuralnet/beacon.h"
-#include "neuralnet/contract/contract.h"
-#include "neuralnet/contract/message.h"
-#include "neuralnet/project.h"
-#include "neuralnet/quorum.h"
-#include "neuralnet/researcher.h"
-#include "neuralnet/tally.h"
-#include "backup.h"
-#include "appcache.h"
-#include "contract/rain.h"
-#include "util.h"
+#include "../main.h"
```

Comment:
The project's include path is set up so that we don't need to reference parent directories in the `#include` directives. This notation is usually reserved for disambiguation for when a file in a subdirectory has the same name as a file in the parent directory.

Reply:
So includes for these would look like:

```cpp
#include "main.h"
#include "rpc/server.h"
...
```

Reply:
Thanks for the advice. I've adjusted it accordingly.